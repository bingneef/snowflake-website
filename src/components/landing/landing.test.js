import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import Landing from "./landing";

describe("Landing", () => {
  it("renders without error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Landing />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches the snapshot", () => {
    const component = renderer.create(<Landing title="Landing" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
