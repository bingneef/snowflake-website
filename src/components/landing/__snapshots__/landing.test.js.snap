// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing matches the snapshot 1`] = `
<section
  className="hero is-fullheight is-info"
  id="landing"
>
  <div
    className="hero-body"
  >
    <div
      className="container container"
    >
      <img
        className="logo logo"
        src="/static/assets/images/snowflake.svg"
      />
      <h1
        className="title"
      >
        Snowflake Webdesign
      </h1>
      <h2
        className="subtitle"
      >
        For a life full of fluffiness
      </h2>
    </div>
  </div>
  <div
    className="hero-foot has-text-centered"
  >
    <div
      className="ripplePlaceholder"
    >
      <a
        className="button is-success button nextButton"
        onClick={[Function]}
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-arrow-down fa-w-14 "
          data-icon="arrow-down"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M413.1 222.5l22.2 22.2c9.4 9.4 9.4 24.6 0 33.9L241 473c-9.4 9.4-24.6 9.4-33.9 0L12.7 278.6c-9.4-9.4-9.4-24.6 0-33.9l22.2-22.2c9.5-9.5 25-9.3 34.3.4L184 343.4V56c0-13.3 10.7-24 24-24h32c13.3 0 24 10.7 24 24v287.4l114.8-120.5c9.3-9.8 24.8-10 34.3-.4z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </a>
      <div
        className="ripple"
      />
      <div
        className="rippleDelay"
      />
    </div>
  </div>
</section>
`;
